CMakeLists.txt

```diff
@@ -71,7 +71,7 @@ if (OCTOMAP_INCLUDE_DIRS AND OCTOMAP_LIBRARY_DIRS)
   list(GET VERSION_LIST 2 OCTOMAP_PATCH_VERSION)
   add_definitions (-DOCTOMAP_MAJOR_VERSION=${OCTOMAP_MAJOR_VERSION}
     -DOCTOMAP_MINOR_VERSION=${OCTOMAP_MINOR_VERSION}
-    -DOCTOMAP_PATCH_VERSION=${OCTOMAP_PATCH_VERSION})
+    -DOCTOMAP_PATCH_VERSION=${OCTOMAP_PATCH_VERSION} -DHPP_FCL_HAVE_OCTOMAP=1)
```

Comment:
Side comment: those definition should be provided by hpp-fcl.pc. You shouldn't have to worry about them in the CMakeLists.txt

Reply:
I think I defintely have issues with the pc, because it was not added in my case...